spec/v2/index.bs

```diff
@@ -172,15 +172,36 @@ For now, this chapter focuses on asynchronous event processing. It will be expan
 
 #### Context and Assumptions #### {#business-cases-async-events-1-context}
 
-- This business case is triggered by a data recipient if they wish to access a Product Footprint **they could not yet access through the `ListFootprints` API**.
+- This business case is triggered by a data recipient if they wish to access a Product Footprint <strong>they could not yet access through the `ListFootprints` API</strong>.
 - The data recipient and data owner have a mutually agreed upon way to identify products.
 
 #### Workflow #### {#business-cases-async-events-1-workflow}
 
 1. The data recipient sends a `ProductFootprintFragment`  to the `Events Endpoint` of the data owner. The data recipient should always include 1 or more product ids in property `productIds`. The data recipient should limit each fragment to exactly 1 specific product (e.g. a specific type of apple instead of all apples that somebody is offering)
-    1. If the recipient is requesting a specific reporting period, it should include the reporting period accordingly (example …)
-    2. if the recipient is requesting a specific geography, it should include …
-    3. if the recipient has another need that is not covered above, the data recipient should <…>
+    1. If the recipient is requesting a specific reporting period, it should include the reporting period accordingly
+        <div class="example">
+          ```json
+          {
+            "productIds": ["urn:pathfinder:company:customcode:buyer-assigned:4321"],
+            "referencePeriodStart": "2023-01-01T00:00:00Z",
+            "referencePeriodEnd": "2023-12-31T23:59:59Z"
```

Comment:
```suggestion
            "referencePeriodEnd": "2024-01-01T00:00:00Z"
```

this as `referencePeriodEnd` is the end of time boundary **excluding**.

Reply:
Thank you, Martin!